homework/list.js

```diff
@@ -1,44 +1,49 @@
 'use strict';
 
-(function() {
+(function(exports) {
+var ListManager = function(){
+	this._listNoteContent = [];
+	this._wrapper = null;	
+};
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
-
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
-
-  function onNoteOpen(event) {
+ListManager.prototype = {
+	
+  start() {
+	this._wrapper = document.querySelector('#note-list-wrapper');
+    this.fetchList()
+	.then((function(data) {
```

Comment:
Okay, good. You have successfully "Promisified" it and use it with another chain.